imap/ctl_conversationsdb.c

```diff
@@ -702,6 +719,25 @@ static int fix_modseqs(struct conversations_state *a,
                 return r;
             }
         }
+        if (ca.key[0] == 'G') {
+            // basecid might be different on the old record, so if they're both v3, then copy the data over
+            if (ca.datalen >= 32 && cb.datalen >= 32 && ca.data[0] == (char)0x83 && cb.data[0] == (char)0x83) {
```

Comment:
What's this 0x83 magic number? Looks like `'S'`?

Reply:
yeah, it's a bit magic.  The version number `if (c & 0x80) version = c & 0x7f`.  I just combined the two.  Sadly the code in conversations.db isn't neatly split into a parser, so it was easier to just hardcode.

Reply:
I've added a comment